owncloudApp/src/main/java/com/owncloud/android/operations/RenameFileOperation.java

```diff
@@ -112,7 +112,7 @@ protected RemoteOperationResult run(OwnCloudClient client) {
             
         } catch (IOException e) {
             Log_OC.e(TAG, "Rename " + mFile.getRemotePath() + " to " + ((mNewRemotePath==null) ?
-                    mNewName : mNewRemotePath) + ": " + result.getLogMessage(), e);
+                    mNewName : mNewRemotePath) + " failed", e);
```

Comment:
It makes sense to prevent form crash. -absolute-
But to see the reason makes maybe sense ? What's about this ?

` mNewName : mNewRemotePath) + ": " + (result==null ? "failed" : result.getLogMessage()), e);`

Reply:
As far as I can tell, `result` will **always** be `null` on line 115. It's assigned `null` at the start of the the method, then only gets assigned a value on line 102. The exception will happen on or above line 102, I believe.

Reply:
Indeed L77 `RemoteOperationResult result = null;` you are right.
And I like Kotlin, then you have to handle it
Thank you